hydra/core/utils.py

```diff
@@ -172,7 +199,8 @@ def run_job(
         return ret
     finally:
         HydraConfig.instance().cfg = orig_hydra_cfg
-        os.chdir(old_cwd)
+        if _chdir:
+            os.chdir(old_cwd)
```

Comment:
Looking back, I think it'd be best to include a test for this change. I'll be happy to write the test since it was my idea to wrap an `if`-block around the call to `os.chdir` here. I'm planning to base the test on the example I gave in my previous comments.

Reply:
I will add a test!  i shoulda done that! :)

Reply:
added a test in 423a761

Reply:
per offline discussion added another  case in 9eb71d3dc9161d56c4895dd02ebdc2ea79594059